docs/configuration/source/keptn.md

```diff
@@ -0,0 +1,37 @@
+---
+id: keptn
+title: Keptn
+sidebar_position: 4
+---
+
+The Botkube Keptn source plugin allows you to consume events from Keptn deployment and notify in configured platforms.
+
+The Keptn plugin is hosted by the official Botkube plugin repository. To enable the Keptn plugin, make sure that the `botkube` repository is defined under `plugins` in the [values.yaml](https://github.com/kubeshop/botkube/blob/main/helm/botkube/values.yaml) file.
+
+```yaml
+plugins:
+  repositories:
+    botkube:
+      url: https://github.com/kubeshop/botkube/releases/download/v1.2.0/plugins-index.yaml
+```
+
+## Enabling plugin
+
+To enable Keptn plugin, add `--set 'sources.keptn.botkube/keptn.enabled=true'` to a given Botkube [`install` command](../../cli/commands/botkube_install.md).
```

Comment:
Not sure if it should be like that (looks a little bit odd)
<img width="967" alt="Screenshot 2023-08-03 at 09 56 17" src="https://github.com/kubeshop/botkube-docs/assets/14016055/48cacf24-1558-4f90-a620-66659ea9d610">

Maybe just that [install command] without "codeblock"

Reply:
It seems we did this same for all executors, it is ok for you if I create a separate pr for that? I need to ask Mateus and Pawel too